appveyor.yml

```diff
@@ -0,0 +1,12 @@
+install:
+- ps: Start-FileDownload https://static.rust-lang.org/dist/$env:PLATFORM.exe -FileName rust.exe
+- rust.exe /VERYSILENT /NORESTART /DIR="C:\Program Files\%PLATFORM%"
+- SET PATH=%PATH%;C:\Program Files\%PLATFORM%\bin
+- rustc -vV
+- cargo -V
+platform:
+- rust-1.1.0-i686-pc-windows-gnu
+# To build against multiple rust versions add them here
```

Comment:
Ok I marked the build as allowed to fail just in case, but it is building (using VS 2014).